beekeeper-integration-tests/src/test/java/com/expediagroup/beekeeper/integration/BeekeeperExpiredMetadataSchedulerApiaryIntegrationTest.java

```diff
@@ -162,11 +164,13 @@ public void expiredMetadataMultipleAddPartitionEvents() throws SQLException, IOE
     amazonSQS.sendMessage(sendMessageRequest(addPartitionSqsMessage.getFormattedString()));
     amazonSQS.sendMessage(sendMessageRequest(addPartitionSqsMessage2.getFormattedString()));
 
-    await().atMost(TIMEOUT, TimeUnit.SECONDS).until(() -> getExpiredMetadataRowCount() == 2);
+    insertExpiredMetadata("s3://location", null);
+
+    await().atMost(60, TimeUnit.SECONDS).until(() -> getExpiredMetadataRowCount() == 3);
 
     List<HousekeepingMetadata> expiredMetadata = getExpiredMetadata();
-    assertExpiredMetadata(expiredMetadata.get(0), LOCATION_A, PARTITION_A_NAME);
-    assertExpiredMetadata(expiredMetadata.get(1), LOCATION_B, PARTITION_B_NAME);
+    assertExpiredMetadata(expiredMetadata.get(1), LOCATION_A, PARTITION_A_NAME);
```

Comment:
same here we get 3 items now?

Reply:
maybe add a assertThat(list.getSize(), is(3));??

Reply:
same as above - have inserted an entry for the table on line 167

Reply:
An assert would be doing the same as line 169, since it waits till theres 3 rows

Reply:
would it make sense to still just assert the get(0) so it's clear what's in the list. It now looks like the test is wrong

Reply:
Yeah I can add that to make it clearer